db/static_data/languages.yml

```diff
@@ -0,0 +1,620 @@
+AA:
+  name: Afar
+  iso: AA
+  yoksis_code:
```

Comment:
`Unit` oluştururken eğitim dili için `UnitInstructionLanguage` modeli referans alınmaya devam edilecek ise başka bir yerde(modelde) dillerle ilgili yöksis bağımlılığımız da yoksa ikinci defa diller için `yoksis_code` tutmaya gerek olmamalı. 
Kısaca, burada `yoksis_code` kullanma mantığını anlayamadım.

Reply:
Evet bu benim gözümden kaçmış.

@isubas Hem Language hem de UnitInstructionLanguage modeli olmamalı. Language daha uygun sanki. UnitInstructionLanguage modelini kaldırabiliriz. Oradaki yoksis_code'ları da senin modele taşırız. Bu durumda Unit modeli

`belongs_to :unit_instruction_type, optional: true`

yerine

`belongs_to :language, optional: true`

gibi değişmeli.

Reply:
Dersler için gerekiyor. Yoksis servisleri baktığımda dersler için test servisler açtıkları ve dersin dilini referans bir tabloya göre aldıkları gördüm. Gelecekte büyük olasılıkla yoksis dersleri içinde bir servis sunacak. Bu yüzden şimdilik `yoksis_code` var bu kod referans tablosundaki veriler için, gelecekte bu yöndeki ihtiyaçları karşılamazsa veya işleri karışıklaştırırsa gerekli değişikliği yaparız.

Reply:
>  Gelecekte büyük olasılıkla yoksis dersleri içinde bir servis sunacak

O zaman ileriye dönük iyi bir yatırım olmuş olur :+1: